static/scripts/rewards/web3/erc721-permit.ts

```diff
@@ -31,9 +32,11 @@ export function claimErc721PermitHandler(reward: ERC721Permit) {
     }
 
     buttonController.showLoader();
-    try {
-      const nftContract = new ethers.Contract(reward.tokenAddress, nftRewardAbi, signer);
 
+    const nftContract = new ethers.Contract(reward.tokenAddress, nftRewardAbi, signer);
+    if (!nftContract) return;
```

Comment:
Did you try NFT minting with this line added?

Reply:
I am not able to do that in my env